elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -110,6 +110,13 @@ private static void from8(ChainedTransformationDescriptionBuilder chainedBuilder
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.CUSTOM_EVIDENCE_DECODER));
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.AGGREGATE_EVIDENCE_DECODER));
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.CERTIFICATE_AUTHORITY));
+        builder.addChildResource(PathElement.pathElement(ElytronDescriptionConstants.SYSLOG_AUDIT_LOG))
+                .getAttributeBuilder()
+                .addRejectCheck(RejectAttributeChecker.DEFINED, ElytronDescriptionConstants.SYSLOG_FORMAT)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, ElytronDescriptionConstants.SYSLOG_FORMAT)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, ElytronDescriptionConstants.RECONNECT_ATTEMPTS)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, ElytronDescriptionConstants.RECONNECT_ATTEMPTS)
```

Comment:
Might need to take the implied value in the old server into account

Reply:
@justinmcook I think what Kabir means here is that if the old behavior was similar the same as {{0}} or {{-1}} we should transform the value to that value if undefined.

Reply:
Updated it to change the undefined value to 0